app/controllers/synthetic_category_controller.rb

```diff
@@ -0,0 +1,47 @@
+# Only a 'show' action
+#
+# Based roughly off of Sufia/CurationConcerns CollectionController#show
+# TODO link to CollectionControllerBehavior in both
+class SyntheticCategoryController < ApplicationController
+    # include Blacklight::AccessControls::Catalog
+    include Blacklight::Base
+
+    # ??
+    copy_blacklight_config_from(::CatalogController)
+
+    def show
+      unless synthetic_category
+        raise ActionController::RoutingError.new("No SyntheticCategory matches `#{params[:id]}`")
+      end
+      set_response # trigger setting of @response, which I think some partials need.
+    end
+
+
+    protected
+
+    # Get controller to find templates in CollectionController too,
+    # so we can re-use more of the sufia stuff for collection. Messy messy.
+    def self.local_prefixes
```

Comment:
So if we're okay postponing the integration test... ready to merge?  I just need to add a comment about the `slug`?